### ISSUE TITLE
Please, add log of gradle task

### ISSUE THREAD

**Issue by Kolyall** [wakatime/jetbrains-wakatime#106]
Hello

is it possible to log gradle tasks which is run in the Jetbrains IDE?

Like it is doing here [https://github.com/passy/build-time-tracker-plugin](https://github.com/passy/build-time-tracker-plugin) ?

**Comment by alanhamlett** [comment]
Yes! It's already on the roadmap, since support for tracking build times was recently added for the WakaTime Xcode plugin. Let's leave this issue open until it's ready to use.

**Comment by gandarez** [comment]
@alanhamlett is this issue done?

**Comment by gandarez** [comment]
@alanhamlett any call out here?

**Comment by alanhamlett** [comment]
9a2a014eac39d5182cc904793e7860d1d73db943 added support for tracking build times, but we can't release it until making `com.intellij.modules.java` an optional dependency to support non-Java IDEs such as PyCharm.
